src/api/app.py

```diff
@@ -60,6 +60,20 @@ def get_semesters():
     semesters, error = class_info.get_semesters()
     return jsonify(semesters) if not error else Response(error, status=500)
 
+@app.route('/api/defaultsemester', methods=['GET'])
+def get_defaultSemester():
+    semester, error = admin_info.get_semester_default()
+    return jsonify(semester) if not error else Response(error, status=500)
+
+@app.route('/api/defaultsemesterset', methods=['POST'])
+def set_defaultSemester():
+    info = request.get_json()
+    success, error = admin_info.set_semester_default(info['default'])
+    if success:
+        return Response(status=200)
+    else:
+        print(error)
+        return Response(error.__str__(), status=500)
```

Comment:
Is this new code setting the default semester or allowing it to be changed? I want to understand what is happening better.

Reply:
The bottom route is calling set_semester_default with the given default semester in order to set the default semester. 

The top rout simply retrieves the current default semester